src/main/windows/callsWidgetWindow.ts

```diff
@@ -233,16 +233,31 @@ export class CallsWidgetWindow {
         ev.preventDefault();
     };
 
+    private toggleWidgetVisibility = (flag: boolean) => {
```

Comment:
Another: `flag` has little meaning... Anything else would be better, imo.  `highIndex` `highVisibility` etc.?

Reply:
@cpoile Interestingly, I copied that arg name straight from the Electron API for one of those methods :) I'll see what I can do.